ui/base-tool/src/main/res/layout/tool_content_text.xml

```diff
@@ -12,6 +12,8 @@
         android:layout_marginRight="@dimen/tool_content_margin_horizontal"
         android:drawableStart="@{model.startImage}"
         android:drawableEnd="@{model.endImage}"
+        android:drawablePadding="8dp"
```

Comment:
@frett I wasn't current if I should make this a const value or not. Let me know and I can change it.

Reply:
I think leaving this here is fine, it's not really being used in multiple places